docs/usage/getting-started/running.md

```diff
@@ -19,6 +19,8 @@ When self-hosting Renovate you're the "administrator" of the bot, this means you
 
 If you're self-hosting Renovate on Windows, read [Self-hosting on Windows](./installing-onboarding.md#self-hosting-on-windows) to prevent line endings from confusing Renovate bot.
 
+If you're running Renovate Community Edition or Renovate Enterprise Edition, refer to the documentation on the public GitHub repository [`mend/renovate-ce-ee`](https://github.com/mend/renovate-ce-ee).
```

Comment:
```suggestion
If you're running Renovate Community Edition or Renovate Enterprise Edition, refer to the documentation on the [`mend/renovate-ce-ee` GitHub repository ](https://github.com/mend/renovate-ce-ee).
```

We can drop the word `public`. The reader will only expect public repositories anyway. 😉

Plus I'm rewriting the link fragment so it's clear where you'll end up.